internal/ingress/status/status.go

```diff
@@ -185,7 +185,7 @@ func NewStatusSyncer(config Config) Sync {
 			go st.syncQueue.Run(time.Second, stop)
 			wait.PollUntil(updateInterval, func() (bool, error) {
 				// send a dummy object to the queue to force a sync
-				st.syncQueue.Enqueue("sync status")
```

Comment:
What's happening currently here? Runtime error because of an incorrect type?

Reply:
I see that it is an interface, so eventually it will get passed to `cache.DeletionHandlingMetaNamespaceKeyFunc` to generate a key. And https://github.com/kubernetes/client-go/blob/8aceb98010c1c18b6b54a35b52fd5b46905e3d7f/tools/cache/store.go#L77 will make sure the key is `"sync status"`